# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-flexbox/dynamic-baseline-change-nested.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-flexbox/dynamic-baseline-change-nested.html.ini
--- a/testing/web-platform/meta/css/css-flexbox/dynamic-baseline-change-nested.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/css-flexbox/dynamic-baseline-change-nested.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,8 +1,9 @@
 [dynamic-baseline-change-nested.html]
   expected:
-    if (os == "linux") and not debug and not fission: ["FAIL", "PASS"]
-    if (os == "linux") and debug and fission: ["FAIL", "PASS"]
-    if (os == "mac") and debug and webrender: ["FAIL", "PASS"]
-    if (os == "mac") and not debug: ["PASS", "FAIL"]
+    if (os == "linux") and not swgl and webrender and not debug and not fission: ["FAIL", "PASS"]
+    if (os == "linux") and not swgl and not webrender and not debug: ["FAIL", "PASS"]
+    if (os == "linux") and not swgl and webrender and debug: ["FAIL", "PASS"]
+    if (os == "mac") and debug and webrender: ["PASS", "FAIL"]
+    if (os == "mac") and not debug: ["FAIL", "PASS"]
     if os == "android": PASS
     FAIL